ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java

```diff
@@ -519,7 +519,7 @@ public String getValue( JsJob job ) {
     // table.addColumn(idColumn, "ID");
     table.addColumn( nameColumn, Messages.getString( "scheduleName" ) );
     table.addColumn( scheduleColumn, Messages.getString( "recurrence" ) );
-    table.addColumn(executableType, Messages.getString( "executableType" ) );
+    table.addColumn(type, Messages.getString( "Type" ) );
```

Comment:
Is this change necessary?  Doesn't do any harm, but curious why it's needed, especially if we're changing the column heading in the UI.

Reply:
We are changing the column header to match with PDI, in PDI its showing as Type, so to match in PUC I have updated.